.github/workflows/e2e.yml

```diff
@@ -25,5 +25,8 @@ jobs:
     - name: Set netfilter conntrack max
       run: sudo sysctl -w net.netfilter.nf_conntrack_max=131072
 
+    - name: Run K8s upgrade e2e tests
```

Comment:
We don't want to run upgrade test with every PR commit - it should be run as part of run-all-suite (#584 ). Once we are sure this test passes, let's move it along with the rest of e2e tests workflow.

Reply:
Okay. Minor version upgrade worked in e2e test workflow so this test has been from this workflow. This test can be run with other PRs after #584 is completed.